### ISSUE TITLE
Doc: Document how to generate and store a `nodekey`

### ISSUE THREAD

**Issue by D4nte** [status-im/nim-waku#847]
The `wakunode2` binary enables passing a `--nodekey` to be used for the node's crypto (e.g. peer id).

However, there is no documentation on how one should generate, store and pass such node key.

Could you please document this?

**Comment by jm-clius** [comment]
I agree, I think this should be done as part of the documentation for the operator trial: https://github.com/status-im/nim-waku/issues/828

**Comment by dao** [comment]
Could you please add the option to pass node key as a file? 

We are moving from docker compose to docker stack for swarm secret support, which mounts secrets to a ramdisk. While i could add a shell script to interpolate this into the args or config file, I'd rather not as it adds otherwise unnecessary complexity to our configuration.
